### DISCUSSION TITLE
实体类继承  Entity<Long>和继承 Entity有什么区别？

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#282] [02.提问]
请教下 有个疑问 实体类继承  Entity<Long>和继承 Entity有什么区别？
带了<Long> 意味着什么？
<img width="492" alt="image" src="https://github.com/dromara/lamp-cloud/assets/6092525/bcddd45a-c10f-4ee1-9cd0-f0f81c05d869">

**Comment by zuihou** [comment]
泛型，表示子类id的类型

**Reply by zqb666** [reply]
updatedBy  createdBy 应该也会影响吧？

**Reply by zuihou** [reply]
是
